pandas/core/frame.py

```diff
@@ -2903,8 +2903,76 @@ def to_html(
         %(returns)s
         See Also
         --------
+        Styler.to_html : Render a DataFrame to HTML with conditional formatting.
         to_string : Convert DataFrame to a string.
-        """
+
+        Notes
+        -----
+        As of version 1.4.0, a warning is shown for the following use of keyword
+        arguments if they are non-default values, in order to align this function with
+        the formatting structure akin to ``Styler.to_html``.
+
+          - ``classes``, likely replaced by ``table_attributes``.
+          - ``sparsify``, likely separated to ``sparse_index``, ``sparse_columns``.
+          - ``max_cols``, likely replaced with ``max_columns`` consistently.
+          - ``formatters``, likely replaced with ``formatter``.
+          - ``float_format``, likely replaced by ``precision``, ``decimal``,
+            ``thousands``.
+          - ``border``, likely removed due to deprecated HTML specification.
+          - ``col_space``, likely removed in favour of CSS solutions.
+          - ``justify``, likely removed in favour of CSS solutions.
+          - ``render_links``, likely removed due to limited functionality.
```

Comment:
I find this actually quite useful, if you have a DataFrame with urls and want to use it in a notebook setting to have clickable links.

Reply:
this was not included in Styler, but I have added a separate PR for it #45058. Called it `hyperlinks` instead of `render_links`.